### ISSUE TITLE
Compile error: Can't find project or library

### ISSUE THREAD

**Issue by scipio314** [scipio314/draftKnight#5]
[Image of error](http://i.imgur.com/gCj4rJ6.png). - reported by [/u/zer0balance](https://www.reddit.com/user/zer0balance)

**Comment by drew3770** [comment]
Had this error too. How can I overcome this and get this to work?

**Comment by scipio314** [comment]
I haven't been able to verify a solution. So if anyone has one please post. You can try the following:  

I think what you need to do is add the correct references in Excel for your project to work. Here is how to do that:  

1) Hit Alt + F11.  
2) Go to "Tools" in the top ribbon.  
3) Look at this [picture](http://i.imgur.com/v26BABZ.png) and make sure all these boxes are checked. You may have to scroll down to find them all.  

I'm on Excel 2010, by default later versions of Excel use different object libraries. I think it should just be a matter of adding an older version of excel's references. Let me know if this works.

**Comment by drew3770** [comment]
I have solver but not open solver. I’ll have to get open solver when I have time.
[cid:image001.png@01D256BE.05BBBB90]


Drew Vrba
Allied Recruiting Consultant
AMN Healthcare
NYSE: AMN
Direct Line: (866)217-8409
Internal Extension: (5596)
Main Line: (800)788-4815
8840 Cypress Waters BLVD, Suite 300, Dallas, TX 75019
Drew.Vrba@medtravelers.com<mailto:Drew.Vrba@medtravelers.com>
[cid:image002.png@01D16A54.4772FB90]https://www.linkedin.com/in/andrew-vrba-43361187
[cid:image004.jpg@01D0F60C.F7A6AB40]

**Comment by KrazyEyes125** [comment]
Is this project still live? I hope so because I love what has been shown here.

**Comment by KrazyEyes125** [comment]
Mine started working after I tried opening the "Tools" option and making sure all boxes were checked that you said and none others. Everything was as it was supposed to be and when I tried running it again it worked somehow.

**Comment by scipio314** [comment]
@KrazyEyes125 yeah this project is still alive. I haven't made any updates in a while but the spreadsheet should (mostly) work. There are a couple quirks that need a fix.
